packages/popup-component/src/components/popup/popup.scss

```diff
@@ -0,0 +1,97 @@
+@import "shared.scss";
+
+.close {
+  color: #000;
```

Comment:
If we have variables _(as I see below)_ use variables instead.

Reply:
This was copied I think from the other file, also we have a lot of dependencies to bootstrap. I just copied these dependencies over for now but I would like to check a better way to do it, im open to suggestions